contracts/mocks/AirdropCampaignDataUpgrade.sol

```diff
@@ -0,0 +1,134 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.15;
+
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+
+/// @title AirdropCampaignData - Data contract for storing of daily hashes of airbro Campaigns
+contract AirdropCampaignDataUpgrade is Initializable {
```

Comment:
Recommended way is to use UUPS upgradeable contracts
https://docs.openzeppelin.com/contracts/4.x/api/proxy#UUPSUpgradeable

And for deploying
```
await upgrades.deployProxy(ContractFactory, [args], {
    initializer: "initialize",
    kind: "uups"
  });
```